src/main/java/com/salesforce/cdp/queryservice/util/QueryExecutor.java

```diff
@@ -46,23 +46,26 @@ public class QueryExecutor extends QueryTokenExecutor {
     }
 
     private final OkHttpClient queryClient;
-
     public QueryExecutor(QueryServiceConnection connection) {
-        this(connection, null, null);
+        this(connection, null, null, false);
```

Comment:
This can directly call the constructor with all the arguments, rather than chaining it.